grpc-native/src/main/java/org/ballerinalang/net/grpc/protobuf/cmd/GrpcCmd.java

```diff
@@ -121,6 +122,12 @@ public void execute() {
             outStream.println(errorMessage);
             return;
         }
+        // Temporary disabled due to new service changes.
```

Comment:
Created an issue to track this https://github.com/ballerina-platform/ballerina-standard-library/issues/729